week-2/D-predicates/exercise.js

```diff
@@ -7,12 +7,14 @@
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
 function isNegative(number) {
-
+  return number < 0;
 }
 
 // Finish the predicate function to test if the passed number is between 0 and 10
 function isBetweenZeroAnd10(number) {
+  return 0<=10;
```

Comment:
This will always return `true` (zero is always "less than or equal" to 10). You need to check two things. Do you remember the operator for "and"? It's `&&`. So `a && b` is true if `a` is true and `b` is true.